cmd/awscollector/copy_docker_binary.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+BINARY=aoc_linux_x86_64
```

Comment:
Small nit I would probably output the same `aoc_linux` binary into a folder with the platform, just to reduce moving parts (renaming files).

Even better could be just `awscollector` perhaps since that's what we end up renaming to anyways. Results in less names to deal with.

Reply:
Yeah that was for backward compatibility in the make file. Any sort of backwards not compatible change we should have a meeting just to discuss that.  Or I can just make more make commands.

Reply:
https://github.com/aws-observability/aws-otel-collector/issues/880